### ISSUE TITLE
No review of pull requests!

### ISSUE THREAD

**Issue by Habacef** [Cardano-NFTs/policyIDs#1663]
Hello? Is anyone working on this project?

**Comment by FufuBcn** [comment]
Seems not... my pull request is open since 8 days !

**Comment by Slatters** [comment]
does anyone know who is the last person to get verified? or is it randomly picked?

**Comment by cardanomushrooms** [comment]
Seems like it is just happening random or projects of interest.

**Comment by santonode** [comment]
So much for decentralization! Pretty sad. I have offered to help them, but I get no response.

**Comment by aiexpression** [comment]
We trying to reach out also, no response yet. waiting for 9 days.

**Comment by Slatters** [comment]
They really think they can keep up with the growth of nfts with just one or two people verifing! this could even be done as a team.
I understant they are trying to keep everything legit and protected! i respect that!

**Comment by JakeAdlerWriter** [comment]
Maybe it's time to get in touch with Charles H and make him aware of this issue re: NFT's.  He won't be happy to hear it I'm sure.

**Comment by KosmosKarl** [comment]
Yes, same issue here I have been waiting for 8 days and no response.

**Comment by NuRealDrawings** [comment]
11 Days ago for me

**Comment by skilltestz** [comment]
10days ago too
https://github.com/Cardano-NFTs/policyIDs/pull/1620

Just update policy only...
